src/main/java/frc/robot/subsystems/BallPath/Shooter/PIDShooterImpl.java

```diff
@@ -100,29 +100,29 @@ public void setShotPosition(ShotPosition shotPosition) {
     public double getSetpointHood(double distance){
         double hoodDif, distDif, difFromUpper, percentToAdd, amountToAdd;
         double returnAmount = 0;
-        Integer Fender = 100_000;
-        Integer Tarmac = 230_000;
-        Integer CloseLaunchPad = 300_000; // pls test
-        Integer FarLaunchPad = 330_000; // pls test
-        Integer HumanPlayer = 400_000; // pls test
-        double[] Distances = {55.0, 153.0, 202.95, 244.77, 305.66};
-        Integer[] HoodValues = {100_000, 230_000, 300_000, 330_000, 400_000};
+        // Integer Fender = 100_000;
+        // Integer Tarmac = 230_000;
+        // Integer CloseLaunchPad = 300_000; // pls test
+        // Integer FarLaunchPad = 330_000; // pls test
+        // Integer HumanPlayer = 400_000; // pls test
+        double[] distances = {55.0, 153.0, 202.95, 244.77, 305.66};
+        Integer[] hoodValues = {100_000, 230_000, 300_000, 330_000, 400_000};
 
         // LinkedHashMap<Double, Integer> foundValues = new LinkedHashMap<>();
         // foundValues.put(55.0, Fender);
         // foundValues.put(153.0, Tarmac);
         // foundValues.put(202.95, CloseLaunchPad);
         // foundValues.put(244.77, FarLaunchPad);
         // foundValues.put(305.66, HumanPlayer);
-        int i = 0;
-        for (Double key : Distances) {
-            if(distance < key){
-                distDif = Distances[i] - Distances[i-1];
-                hoodDif = HoodValues[i] - HoodValues[i-1];
-                difFromUpper = Distances[i] - distance;
+        for (int i = 0; i < distances.length; i++) {
+            double key = distances[i];
+            if(distance < key && i != 0){
```

Comment:
Rather than ignoring the `i == 0` case, why not just start the loop at `i = 1` on the first iteration?